### ISSUE TITLE
Tab text doesn't match pages

### ISSUE THREAD

**Issue by ACharbonneau** [nih-cfde/cfde-deriva#192]
This is the splash page of the portal:
![image](https://user-images.githubusercontent.com/1719360/112902066-98adba00-90b3-11eb-83aa-845eb6003f65.png)

It seems like my tab should read 'home' or similar, not 'dashboard'

**Comment by karlcz** [comment]
I think this is just the HTML page `title` being set in the static site.  
```
<html lang="en">
  <head>
    <title>CFDE Dashboard</title>
...
```

**Comment by ACharbonneau** [comment]
For the moment, lets go with `CFDE Data Review` and open an issue for the next cycle to see if we can get the name and catalog id idea working.

**Comment by jonathancrabtree** [comment]
Done, included in CFDE-20210329.3
